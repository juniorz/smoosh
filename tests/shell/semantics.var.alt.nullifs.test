IFS=
printf '<%s>\n' ${x+uhoh} a b

f() { echo $#; printf '<%s>\n' "$@" ; }
f ${x+uhoh} a b

x=hi
f ${x+uhoh} a b

