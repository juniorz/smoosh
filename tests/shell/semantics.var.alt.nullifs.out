<a>
<b>
2
<a>
<b>
3
<uhoh>
<a>
<b>
